time/src/test/java/io/spine/time/TimeChangesTest.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2018, TeamDev. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.time;
+
+import com.google.common.testing.NullPointerTester;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import static io.spine.test.Tests.assertHasPrivateParameterlessCtor;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class TimeChangesTest {
+
+    @Test
+    void utilityCtor() {
+        assertHasPrivateParameterlessCtor(TimeChanges.class);
+    }
+
+    @Test
+    void nullCheck() {
+        ZoneOffset utc = ZoneOffsets.utc();
+        new NullPointerTester()
+                .setDefault(OffsetTime.class, OffsetTimes.now(utc))
+                .setDefault(OffsetDateTime.class, OffsetDateTimes.now(utc))
+                .setDefault(LocalDate.class, LocalDates.now())
+                .setDefault(LocalTime.class, LocalTimes.now())
+                .testAllPublicStaticMethods(TimeChanges.class);
+    }
+
+    @Nested
+    @DisplayName("fail to create a change for equal values")
+    class RejectEqual {
+
+        @Test
+        @DisplayName("LocalDate")
+        void localDates() {
+            final LocalDate today = LocalDates.now();
```

Comment:
Made me think `today()` can a better alternative to `now()` in this class.

Reply:
It follows the naming from Java Time.